pep-0001.txt

```diff
@@ -80,15 +80,15 @@ Python's Steering Council
 
 There are several references in this PEP to the "Steering Council" or "Council".
 This refers to the current members of the elected Steering Council described
-in PEP 13 [5]_, in their role as the final authorities on whether or not PEPs
+in :pep:`13`, in their role as the final authorities on whether or not PEPs
```

Comment:
I thought docutils already automatically parsed `RFC \d+` and `PEP \d+` to make links!

Reply:
See the PR description:

> This PR seeks to disable the Docutils inliner for PEP and RFC references. The inliner looks for patterns of `pep-\d+(.txt)?` or `PEP\s+\d+` for PEPs, and `RFC(-|\s+)?\d+` for RFCs. It requires mokey-patching in the implementation and picks up text that isn't intended to be an explicit reference, for example mailing list thread titles, or branch names with a `pep-nnn` pattern.
> 
> To do this in the most backwards-compatible requires changing all PEP nnnn text to `` :pep:`nnnn` `` (similar for RFCs). The `:pep:` role is supported in plain Docutils, and is currently used in several PEPs, so will work with the current `pep2html` based system during the transition period.

Reply:
Another issue is the Inliner is really hard to patch and is even more complicated to explain/understand than Docutils roles -- for future maintence it would be simpler to have one thing implementing PEP / RFC / BCP / BPO / ... references

A